stampifier.py

```diff
@@ -84,6 +125,13 @@ def _summarize(self):
 
         self.stampified_pages = summarizer.get_summarized_content()
 
+    def _order_stamp_pages(self):
```

Comment:
Is this change intentional? Revert if else.

Reply:
Done! It was not intentional. It was because of some misplacement while moving functionalities from one file to another.